docs/develop/which-tools.mdx

```diff
@@ -38,37 +38,37 @@ Terra-Development-Suite
 
 ## Terrain <ThemedImage sources={{light:"/img/icons/terrain.svg", dark:"/img/icons/dark/terrain.svg"}} height="30px" />
 
-Terra’s all-in-one development environment, Terrain provides front-end and smart contract scaffolding for your dApp. Get started in 5 minutes with the [quickstart guide](./get-started.mdx).
+Terra’s all-in-one development environment, Terrain provides front end and smart contract scaffolding for your dApp. Get started in 5 minutes with the [quickstart guide](./get-started.mdx).
```

Comment:
I think this should be "front-end" because its an adjective (front-end scaffolding)